src/pyradiosky/skymodel.py

```diff
@@ -2447,6 +2455,9 @@ def _calc_average_rotation_matrix(self):
             )
             axes_altaz = axes_icrs.transform_to("altaz")
         else:
+            # can only get here if we've already checked that lunarsky is installed
+            from lunarsky import SkyCoord as LunarSkyCoord
```

Comment:
We're making this assumption because we are so far into the simulation/calculations that the other functions _should_ have been called? Because I again can't see where the check is with 0 context. With context, we obviously expect this to be called from within a `coherency_calc` right? I just wonder if a user could end up where without the other utilities?

Reply:
I added a check right before this to ensure it's checked even if a user calls this private method directly.